io/plugins/eu.esdihumboldt.hale.io.json.ui/src/eu/esdihumboldt/hale/io/json/ui/util/EnumJSONAutoOrderTypes.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2016 Data Harmonisation Panel
+ * 
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution. If not, see <http://www.gnu.org/licenses/>.
+ * 
+ * Contributors:
+ *     Data Harmonisation Panel <http://www.dhpanel.eu>
+ */
+
+package eu.esdihumboldt.hale.io.json.ui.util;
+
+/**
+ * Enumeration for different types of winding orders for Geometry. Selected type
+ * will be applied geometry. In case of Polygon/MultiPolygon selected type will
+ * be applied to the shell and the reversed type is applied to the holes.
+ * 
+ * @author Arun
+ */
+public enum EnumJSONAutoOrderTypes {
```

Comment:
This replicates the [JsonReadMode](https://github.com/halestudio/hale/blob/master/io/plugins/eu.esdihumboldt.hale.io.json/src/eu/esdihumboldt/hale/io/json/internal/JsonReadMode.java) enum already defined and used by the reader. It should only be defined once.